src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,55 @@
+package seedu.addressbook.data.tag;
+
+import java.util.List;
+
+/**
+ * Class to associate a tag with a tag state and name.
+ */
+public class Tagging {
+    
+    public static enum TagState {
+        ADD("+"), REMOVE("-");
+        
+        private final String symbol;
+        
+        private TagState(String symbol) {
+            this.symbol = symbol;
+        }
+        
+        public String getSymbol() {
+            return symbol;
+        }
+    }
+    
+    private TagState tagState;
+    private String name;
+    private Tag tag;
+    
+    public Tagging(TagState tagState, String name, Tag tag) {
```

Comment:
You must have the associated classes as the arguments in the constructor. So a Person class should be given instead of a String name